### PR TITLE
ruby: print actual execption on user code failure

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]
If tests don't cover the topic, it would be great to always include QA screenshots in the description.

Other than that, LGTM.

**Review by abnegate — APPROVED** [review]

File: runtimes/ruby-3.0/server.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def json(object)
     p e
     status 500
     content_type :json
-    return { code: 500, message: 'File not found or is not a valid ruby file.' }.to_json
+    return { code: 500, message: "Exception Occurred #{e.class}. Message: #{e.message}. Backtrace:  \n #{e.backtrace.join("\n")}" }.to_json
```
Comment (christyjacob4) [inline runtimes/ruby-3.0/server.rb:79]:
```suggestion
    return { code: 500, message: "#{e.backtrace.join("\n")}" }.to_json
```

File: runtimes/ruby-3.1/server.rb [hunk 2]
```diff
@@ -76,7 +76,7 @@ def json(object)
     p e
     status 500
     content_type :json
-    return { code: 500, message: 'File not found or is not a valid ruby file.' }.to_json
+    return { code: 500, message: "Exception Occurred #{e.class}. Message: #{e.message}. Backtrace:  \n #{e.backtrace.join("\n")}" }.to_json
```
Comment (christyjacob4) [inline runtimes/ruby-3.1/server.rb:79]:
```suggestion
    return { code: 500, message: "#{e.backtrace.join("\n")}" }.to_json
```
Just the trace is good enough 👍

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
minor comment

**Pull request merged**